### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'file_size_limit'

### ISSUE THREAD

**Issue by charlieamer** [supabase-community/storage-py#77]
**Describe the bug**
In production environment I get the error in title. In local development however, I don't get that error. This is the stacktrace:

```
Traceback (most recent call last):
  File ...
    bucket = client.storage().get_bucket('tasks')
  File ".../storage3/_sync/bucket.py", line 52, in get_bucket
    return SyncBucket(**json, _client=self._client)
TypeError: __init__() got an unexpected keyword argument 'file_size_limit'
```
I solved this by adding 2 properties to `BaseBucket` in `types.py`:
```python3
file_size_limit: int
allowed_mime_types: list[string]
```

However when I add these 2 lines, the development environment stops working because of new error: `TypeError: __init__() missing 2 required positional arguments: 'file_size_limit' and 'allowed_mime_types'`

**To Reproduce**
```python3
f = open('test.txt', 'w')
f.write("HELLO WORLD")
f.close()
client = create_client(os.environ['SUPABASE_URL'], os.environ['SUPABASE_KEY'])
# create SOME_BUCKET in your supabase
client.storage().get_bucket('SOME_BUCKET').upload('test.txt', 'test.txt')
```

**Expected behavior**
No error

**Screenshots**
/

**Desktop (please complete the following information):**
 - Ubuntu
 - Python 3.9

**Additional context**
/

**Comment by anand2312** [comment]
What version are you on? This was fixed in #72 and released in storage-py v0.5.2

**Comment by anand2312** [comment]
Fixed in storage-py 0.5.2, and added upstream in supabase-py 1.0.2
